README.md

```diff
@@ -92,6 +93,24 @@ The above output includes only software that is visible in the container (i.e.,
 syft <image> --scope all-layers
 ```
 
+#### Format conversion - __experimental__
+
+Syft can convert between different SBOM formats, such as CycloneDX, SPDX, and Syft's own format. The ability to convert SBOMs helps saving the time and effort it would take to recreate SBOMs from container images, or file systems.
```

Comment:
>The ability to convert SBOMs helps saving the time and effort it would take to recreate SBOMs from container images, or file systems.

Suggestion:

The ability to convert existing SBOMs means you can create SBOMs in different formats quickly, without the need to regenerate the SBOM from scratch, which may take significantly more time.